components/PaymentForm.js

```diff
@@ -15,6 +15,7 @@ import { Paragraph } from './Typography';
 import { saveUPIStatus } from '../services/upi';
 
 function PaymentForm({ onClose = () => null }) {
+  const PROCESSING_CHARGE_PERCENT = 2.36;
```

Comment:
EXTRACT this to other constants we have here https://github.com/coderplex/donation-page/blob/master/constants/index.js